src/main/java/com/mercadopago/MercadoPagoConfig.java

```diff
@@ -0,0 +1,176 @@
+package com.mercadopago;
+
+import com.mercadopago.net.IHttpClient;
+import com.mercadopago.net.MPDefaultIHttpClient;
+import java.util.logging.Level;
+import org.apache.http.HttpHost;
+import org.apache.http.client.HttpRequestRetryHandler;
+
+public class MercadoPagoConfig {
+    private static final String DEFAULT_BASE_URL = "https://api.mercadopago.com";
+    private static final String CURRENT_VERSION = "1.11.0";
```

Comment:
Necessário essa versão?
```suggestion
    private static final String CURRENT_VERSION = "2.0.0";
```

Reply:
Realmente não, removido.